src/Bicep.LangServer.UnitTests/BicepExternalSourceRequestHandlerTests.cs

```diff
@@ -247,7 +243,7 @@ public async Task RestoredValidModule_WithSource_RequestingBicepFile_ShouldRetur
 
             var bicepSource = "metadata hi = 'This is the bicep source file'";
             var bicepUri = new Uri("file:///foo/bar/entrypoint.bicep");
-            var sourceArchive = SourceArchive.UnpackFromStream(SourceArchive.PackSourcesIntoStream(bicepUri, new Core.Workspaces.ISourceFile[] {
+            var sourceArchive = SourceArchive.UnpackFromStream(SourceArchive.PackSourcesIntoStream(bicepUri, null, new Core.Workspaces.ISourceFile[] {
```

Comment:
Similar to other comments:  I recommend qualifying `documentLinks: null` to improve readability